collectors/perf.go

```diff
@@ -63,240 +40,79 @@ type PerfCollector struct {
 func NewPerfCollector() *PerfCollector {
 
 	return &PerfCollector{
-		AvgDiskBytesPerRead: prometheus.NewDesc(
-			prometheus.BuildFQName(wmiNamespace, "perf", "avg_disk_bytes_per_read"),
-			"AvgDiskBytesPerRead",
+		QueueLength: prometheus.NewDesc(
+			prometheus.BuildFQName(wmiNamespace, "perf", "queue_length"),
+			"The number of requests outstanding on the disk (LogicalDisk.CurrentDiskQueueLength)",
 			[]string{"volume"},
 			nil,
 		),
 
-		AvgDiskBytesPerRead_Base: prometheus.NewDesc(
-			prometheus.BuildFQName(wmiNamespace, "perf", "avg_disk_bytes_per_read_base"),
-			"AvgDiskBytesPerRead_Base",
+		ReadBytesPerSec: prometheus.NewDesc(
+			prometheus.BuildFQName(wmiNamespace, "perf", "read_bytes_per_sec"),
+			"The rate at which bytes are transferred from the disk during read operations (LogicalDisk.DiskReadBytesPerSec)",
 			[]string{"volume"},
 			nil,
 		),
 
-		AvgDiskBytesPerTransfer: prometheus.NewDesc(
-			prometheus.BuildFQName(wmiNamespace, "perf", "avg_disk_bytes_per_transfer"),
-			"AvgDiskBytesPerTransfer",
+		ReadsPerSec: prometheus.NewDesc(
+			prometheus.BuildFQName(wmiNamespace, "perf", "reads_per_sec"),
+			"The rate of read operations on the disk (LogicalDisk.DiskReadsPerSec)",
```

Comment:
I just found the same thing, but at a different part of msdn (it's a maze of a site...): https://msdn.microsoft.com/en-us/windows/hardware/aa389383(v=vs.71). I'm changing the metrics to counters, now. What would be good new names? `reads_total` etc?